mygame.py

```diff
@@ -67,6 +70,16 @@ def Step(self):
             elif lowcmd in self.npc_commands:
                 for greeting in self.worldmap.greetnpcs(self.player1.position.x, self.player1.position.y):
                     self.graphics.RenderText(greeting)
+            elif lowcmd in self.item_commands:
+                item_get = input("What will you pick up?")
+                item = self.worldmap.pickupitems(self.player1.position.x, self.player1.position.y, item_get)
+                if item == "You cannot get that.":
```

Comment:
so here, if pickupitems returns None, this if statement becomes cleaner:

if item:
  self.player1.additemtoinv(item)
else:
  print("You cannot get that.")  

now you dont repeat that string, and skip a string comparison operation.